Week3/homework/1-step3.js

```diff
@@ -1,11 +1,12 @@
-'use strict';
+"use strict";
 
 function foo(func) {
-  // What to do here? 
+  func();
+  console.log("This line will appear second");
 }
 
```

Comment:
The results of calling `func()` should be returned from the function. `foo` knows nothing about the function it is calling, and cannot make assumptions that it can throw away the return value from `func`.